src/System Application/App/Email/src/Email/EmailImpl.Codeunit.al

```diff
@@ -119,6 +124,127 @@ codeunit 8900 "Email Impl"
         exit(Send(EmailMessage, EmailAccountId, EmailConnector, false, CurrentDateTime(), EmailOutbox));
     end;
 
+    procedure Reply(EmailMessage: Codeunit "Email Message"; ExternalId: Text; EmailAccountId: Guid; EmailConnector: Enum "Email Connector"): Boolean
+    var
+        EmailOutbox: Record "Email Outbox";
+    begin
+        exit(Reply(EmailMessage, ExternalId, EmailAccountId, EmailConnector, true, CurrentDateTime(), false, EmailOutbox));
+    end;
+
+    procedure Reply(EmailMessage: Codeunit "Email Message"; ExternalId: Text; EmailAccountId: Guid; EmailConnector: Enum "Email Connector"; ReplyAll: Boolean): Boolean
+    var
+        EmailOutbox: Record "Email Outbox";
+    begin
+        exit(Reply(EmailMessage, ExternalId, EmailAccountId, EmailConnector, true, CurrentDateTime(), ReplyAll, EmailOutbox));
+    end;
+
+    procedure Reply(EmailMessage: Codeunit "Email Message"; ExternalId: Text; EmailAccountId: Guid; EmailConnector: Enum "Email Connector"; var EmailOutbox: Record "Email Outbox")
+    begin
+        Reply(EmailMessage, ExternalId, EmailAccountId, EmailConnector, false, CurrentDateTime(), false, EmailOutbox);
+    end;
+
+    procedure Reply(EmailMessage: Codeunit "Email Message"; ExternalId: Text; EmailAccountId: Guid; EmailConnector: Enum "Email Connector"; ReplyAll: Boolean; var EmailOutbox: Record "Email Outbox")
+    begin
+        Reply(EmailMessage, ExternalId, EmailAccountId, EmailConnector, false, CurrentDateTime(), ReplyAll, EmailOutbox);
+    end;
+
+    procedure Reply(EmailMessage: Codeunit "Email Message"; ExternalId: Text; EmailAccountId: Guid; EmailConnector: Enum "Email Connector"; InBackground: Boolean; NotBefore: DateTime; ReplyAll: Boolean; var EmailOutbox: Record "Email Outbox"): Boolean
```

Comment:
would it not be nicer if InBackgrouns and NotBefore were at the end? Now the order of params changes with overloads...

Reply:
Moved them. Also renamed the `Reply` functions that just set `ReplyAll` to true. So they're clearer rather than have all of them be `Reply`.